e2e-tests/src/ProjectTest.js

```diff
@@ -77,11 +77,11 @@ export const flow = {
         },
         {
             "type": "navigate",
-            "url": "http://localhost:3000/",
+            "url": "https://deploy-preview-67--sporosdaoapp-dev.netlify.app/",
```

Comment:
The url has to be executable in any local or ci environment. Should not depend on a hardcoded URL.